.github/workflows/pytests.yml

```diff
@@ -13,49 +13,51 @@ concurrency:
 jobs:
   tests:
     runs-on: ubuntu-latest
-    env: 
+    env:
       # https://www.notion.so/nibiru/Resources-and-Repo-Configs-b31aa8074a2b419d80b0c946ed5efab0
       CHAIN_ID: ${{ secrets.CHAIN_ID }}
       HOST: ${{ secrets.HOST }}
       VALIDATOR_MNEMONIC: ${{ secrets.VALIDATOR_MNEMONIC }}
+      ORACLE_MNEMONIC: ${{ secrets.ORACLE_MNEMONIC }}
       GRPC_PORT: ${{ secrets.GRPC_PORT }}
       LCD_PORT: ${{ secrets.LCD_PORT }}
+      NETWORK_INSECURE: ${{ secrets.NETWORK_INSECURE }}
```

Comment:
Why is it a secret and also set in the README? Is that to make it configurable for other networks when we test in CI workflows?

Reply:
It is an env variable set in the .env file of the tests. It's a secret so we can set it up in the github repo if we want to run tests on another chain than the one we currently use